tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -249,4 +249,5 @@ def expm(self, matrix: Tensor) -> Tensor:
     return tf.linalg.expm(matrix)
 
   def jit(self, fun: Callable, *args: List, **kwargs: dict) -> Callable:
-    return tf.function(fun, *args, **kwargs)
+    # tf.function is slow and bad.
```

Comment:
maybe add a comment to the dosctring of BaseCharge explaining that jit only works for jax

Reply:
Good idea.